### ISSUE TITLE
Typescript can't find type declaration for APIError

### ISSUE THREAD

**Issue by richardvanbergen** [payloadcms/payload#251]
# Bug Report

 Simple one this time. I'm trying to use strict mode but `payload/errors` doesn't seem to export any types?

![image](https://user-images.githubusercontent.com/3854592/127416917-4810f167-094b-4732-b7b7-6a9c017fda63.png)

```
Could not find a declaration file for module 'payload/errors'. '/home/richard/code/payload-plugin-s3/node_modules/payload/errors.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/payload` if it exists or add a new declaration (.d.ts) file containing `declare module 'payload/errors';`ts(7016)
```

## Steps to Reproduce
1. import { APIError } from 'payload/errors'
2. See squiggly warning line
3. Enable strict mode to see red squiggly error line

## Detailed Description

Payload version: 0.8.0

**Comment by denolfe** [comment]
Thanks for the report @richardvanbergen . PR created. 

We're any/all issues relate to running in strict mode so keep them coming 👍

**Comment by denolfe** [comment]
This has been released in 0.8.2 🎉
